src/app/indiv-link/indiv-link.component.html

```diff
@@ -13,7 +13,8 @@
         <div class="button-wrapper">
             <button mat-icon-button aria-label="highlight button" 
                 (click)="highlight();toggle()" 
-                [color]="link.highlighted ? 'accent' : 'black'">
+                [disabled]="!link.visible"
+                [color]="link.highlighted ? 'accent' : 'basic'">
```

Comment:
By the way, do we know for sure if the link is not visible (using our current definition of visible), it is not highlightable ?

Reply:
Yes, the visible check isn't perfect, but at least the way it is now, it is only wrong in the other direction. If a link says it's invisible it definitely can't be highlighted.